pkg/integrations/configrequest/handler.go

```diff
@@ -0,0 +1,40 @@
+package configrequest
+
+import (
+	"github.com/newrelic/infrastructure-agent/internal/integrations/v4/integration"
+	"github.com/newrelic/infrastructure-agent/pkg/databind/pkg/databind"
+	"github.com/newrelic/infrastructure-agent/pkg/integrations/configrequest/protocol"
+	"github.com/newrelic/infrastructure-agent/pkg/log"
+)
+
+var (
+	// helper for testing purposes
+	NoopHandleFn = func(configProtocol protocol.ConfigProtocol) {}
+)
+
+type Entry struct {
+	Definition integration.Definition
+	YAMLConfig databind.YAMLConfig
```

Comment:
I find this name a bit confusing. When I see "YAMLConfig", I think of the entire integration config, not only the databind part. Perhaps we could call it "DatabindConfig" or something more explicit